runtime/doc/api.txt

```diff
@@ -2700,6 +2700,8 @@ nvim_buf_set_extmark({buffer}, {ns_id}, {line}, {col}, {*opts})
                     callbacks. The mark will only be used for the current
                     redraw cycle, and not be permantently stored in the
                     buffer.
+                  • invalidate : boolean that indicates whether to remove the
+                    extmark if the start of its range is deleted.
```

Comment:
I don't think we want just the start of the range. I think invalidate should delete the mark only if the entire range is deleted. If just the head or the tail of a range is deleted, then the mark is shortened, like it does already.

Reply:
Yeah agree TBH. I'll try to adjust for that.